Makefile

```diff
@@ -0,0 +1,28 @@
+.PHONY: help install run start test lint format
+
+help: ## Print all commands (default)
+	@grep -E '^[a-zA-Z0-9_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
+
+####### BUILD #######
+
+install: ## Install dependencies
```

Comment:
I cloned the repository, I made sure I had npm installed as in the README and launched `make install` (which isn't documented), it didn't work:

![image](https://user-images.githubusercontent.com/2562270/33181326-20e0abec-d070-11e7-8d52-28ee6c729b72.png)

Reply:
404 explanation: index.js in pages should be in lowercase.